src/server/user_mgmt/base_users.py

```diff
@@ -0,0 +1,66 @@
+from sqlalchemy.sql.schema import ColumnDefault
+from config import engine
+from api import user_api
+import sqlalchemy as sa
+
+from sqlalchemy import Table, Column, Integer, String, MetaData, ForeignKey
+
+metadata = sa.MetaData()
+
+
+def create_base_roles():
+    with engine.connect() as connection:
+        result = connection.execute("select role from pdp_user_roles")
+        role_count = len(result.fetchall())
+        if role_count == 0:
+            connection.execute("INSERT into pdp_user_roles  values (1, 'user') ")
+            connection.execute("INSERT into pdp_user_roles  values (2, 'editor') ")
+            connection.execute("INSERT into pdp_user_roles  values (9, 'admin') ")
+
+        else:
+            print(role_count, "roles already present in DB, not creating")
+
+
+def create_base_users():  # TODO: Just call create_user for each
```

Comment:
Super small, but can we add a flag here or something to separate seed data from test data?  This could be a new task after we merge this branch

Reply:
Prefix base users with `base_` and test users with `test_`?